chat-plugins/mafia.js

```diff
@@ -396,8 +399,31 @@ class MafiaTracker extends Rooms.RoomGame {
 		return true;
 	}
 
-	getHammer() {
-		return Math.floor(Object.keys(this.players).length / 2) + 1;
+	resetHammer() {
+		this.setHammer(Math.floor(Object.keys(this.players).length / 2) + 2);
+	}
+
+	setHammer(count) {
+		this.hammerCount = count;
+		this.sendRoom(`The hammer count at ${this.hammerCount}, and lynches have been reset.`, {declare: true});
+		this.lynches = Object.create(null);
+		this.hasPlurality = null;
+		for (const player of Object.values(this.players)) {
+			player.lynching = '';
+		}
+	}
+
+	shiftHammer(count) {
+		this.hammerCount = count;
+		this.sendRoom(`The hammer count has been shifted to ${this.hammerCount}. Lynches have not been reset.`, {declare: true});
+		let hammered = [];
+		for (const lynch in this.lynches) {
+			if (this.lynches[lynch].count >= this.hammerCount) hammered.push(lynch === 'nolynch' ? 'Nobody' : lynch);
+		}
+		if (hammered.length) {
+			this.sendRoom(`${hammered.length} player${Chat.plural(hammered.length, "s have ", " has ")}been hammered; ${hammered.join(', ')}`, {declare: true});
```

Comment:
Unfortunately for you, this got me thinking about readability, so I pushed a new `Chat.count` function, so this can be:

    `${Chat.count(hammered, "player has", "players have")} been hammered...`

Reply:
Unfortunately^2 for you, I added a new change which will further improve readability:

    `${Chat.count(hammered, "players have")} been hammered...`

Reply:
Also note: `:` not `;`